app/src/main/java/me/saket/cascade/sample/MainActivity.kt

```diff
@@ -93,14 +94,14 @@ class MainActivity : AppCompatActivity() {
 
     return CascadePopupMenu.Styler(
       background = {
-        RoundedRectDrawable(getColor(R.color.popup_background), radius = 8f.dip)
+        RoundedRectDrawable(ResourcesCompat.getColor(resources, R.color.popup_background, theme), radius = 8f.dip)
```

Comment:
Wanna use `ContextCompat` instead that only takes in 2 params instead of 3?

I love how confusing resource lookup has become these days with `ResourcesCompat`, `AppCompatResources` and `ContextCompat`.

Reply:
Actually nevermind, let's merge this right away and I can change this in a follow-up.